patches/net/minecraft/world/entity/LivingEntity.java.patch

```diff
@@ -550,6 +561,15 @@
              this.moveRelative(f1, p_365480_);
              this.move(MoverType.SELF, this.getDeltaMovement());
              Vec3 vec3 = this.getDeltaMovement();
+@@ -2249,7 +_,7 @@
+ 
+             vec3 = vec3.multiply((double)f, 0.8F, (double)f);
+             this.setDeltaMovement(this.getFluidFallingAdjustedMovement(d1, flag, vec3));
+-        } else {
++        } else if (this.isInLava() && this.isAffectedByFluids() && !this.canStandOnFluid(fluidState)) {
```

Comment:
is this isInLava method call supposed to be `(this.isInFluidType(fluidstate) && fluidstate.getFluidType() != net.neoforged.neoforge.common.NeoForgeMod.LAVA_TYPE.value())`?

Reply:
In all honesty I have no idea. I just replicated all the changes [as they used to be in 1.21.1](https://github.com/neoforged/NeoForge/blob/1.21.1/patches/net/minecraft/world/entity/LivingEntity.java.patch#L485-L506) and it works fine. I'll see if its necessary for it to be like this